### ISSUE TITLE
RNN-Transducer Loss support

### ISSUE THREAD

**Issue by Times125** [tensorflow/tensorflow#32727]
I found that there are no RNN-Transducer  loss in the new version. I have to use third party pkg like [wrap-transducer](https://github.com/HawkAaron/warp-transducer/tree/master/tensorflow_binding)
So, official support is therefore expected. THKS！

**Comment by dynamicwebpaige** [comment]
Though this is potentially out of the scope of TensorFlow core, it might make sense to add to [TensorFlow Addons](https://www.github.com/tensorflow/addons) as a loss function. 

Rerouting the request to `addons`; @seanpmorgan for visibility.
